Document/0x05b-Basic-Security_Testing.md

```diff
@@ -752,16 +752,49 @@ Check the [Termux Wiki](https://wiki.termux.com/wiki/Remote_Access "Termux Remot
 
 #### Obtaining and Extracting Apps
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1238>
+There are several ways of extracting apk files from a device. You will need to decide which one is the easiest method depending if the app is public or private.
 
 ##### App Store
 
+One of the easiest option is to download the apk from websites that mirror public applications from the Google Play Store. However, keep in mind that these sites are not offical and there is no guarantee that the application hasn't been repackaged or contain malware. A few reputable websites that host apks and are not known for modifying apps and claim to MD5 checksum the apps are:
+
+- [APKMirror](https://apkmirror.com "APKMirror")
+- [APKPure](https://apkpure.com "APKPure")
+
+Beware that you do not have control over these sites and you can not guarantee what they do in the future. Only use them if it's your only option left.
+
 ##### Recovering the App Package from the Device
 
-###### From Rooted Devices
+Obtaining app packages from the device is the recommended method as we can guarantee the app hasn't been modified.
 
 ###### From Non-Rooted Devices
```

Comment:
the level of depth is just too high unfortunately... Can you restructure this a bit ? After all, you are ending the story with that you do not need a rooted device.